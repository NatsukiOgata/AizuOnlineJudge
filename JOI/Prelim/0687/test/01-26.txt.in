100
213 415 494 582 264 231 561 209 263 179 273 194 280 103 461 220 163 226 149 259 333 104 221 586 517 527 161 120 391 438 434 403 106 314 242 253 551 552 87 566 150 223 201 457 185 58 116 514 386 72 425 229 78 516 323 93 199 455 184 132 123 331 279 535 399 156 473 414 367 515 313 472 547 463 416 262 205 526 478 124 138 131 292 556 109 127 175 420 306 164 600 50 398 204 155 454 500 92 388 505
