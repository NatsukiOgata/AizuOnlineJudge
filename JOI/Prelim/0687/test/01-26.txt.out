27457
2746
